### ISSUE TITLE
Loss function in paper =  backward() into SeparableConvolution.py ?

### ISSUE THREAD

**Issue by adrienpitz** [sniklaus/pytorch-sepconv#9]
Hi ! 

I'm currently trying to retrain your network only with fast moving sport images.

To make the backpropagation, I need to implement the `backward()` function. 

Do I just need to implement the loss function described in the paper section 3.1.1 ?

The question might seem trivial, but I prefer to ask the question :)

Thanks a lot,
Adrien

**Comment by ee16d409** [comment]
Could you please provide the backprop code for "SeparableConvolution.py"? I'm considering to retrain the model for my specific application. It will be very helpful if you provide the backprop code for the custom autograd function implemented in "SeparableConvolution.py".
